tests/testthat/test-shinytest2-tm_t_crosstable.R

```diff
@@ -0,0 +1,107 @@
+app_driver_tm_t_crosstable <- function() {
+  data <- simple_cdisc_data()
+  init_teal_app_driver(
+    data = data,
+    modules = tm_t_crosstable(
+      label = "Cross Table",
+      x = teal.transform::data_extract_spec(
+        dataname = "ADSL",
+        select = teal.transform::select_spec(
+          label = "Select variable:",
+          choices = teal.transform::variable_choices(data[["ADSL"]], subset = function(data) {
+            idx <- !vapply(data, inherits, logical(1), c("Date", "POSIXct", "POSIXlt"))
+            return(names(data)[idx])
+          }),
+          selected = "COUNTRY",
+          multiple = TRUE,
+          ordered = TRUE,
+          fixed = FALSE
+        )
+      ),
+      y = teal.transform::data_extract_spec(
+        dataname = "ADSL",
+        select = teal.transform::select_spec(
+          label = "Select variable:",
+          choices = teal.transform::variable_choices(data[["ADSL"]], subset = function(data) {
+            idx <- vapply(data, is.factor, logical(1))
+            return(names(data)[idx])
+          }),
+          selected = "SEX",
+          multiple = FALSE,
+          fixed = FALSE
+        )
+      ),
+      show_percentage = TRUE,
+      show_total = TRUE,
+      pre_output = NULL,
+      post_output = NULL,
+      basic_table_args = teal.widgets::basic_table_args(
+        subtitles = "Table generated by Crosstable Module"
+      )
+    ),
+    timeout = 3000
+  )
+}
+
+test_that("e2e - tm_t_crosstable: Initializes without errors", {
+  skip_if_too_deep(5)
+  app_driver <- app_driver_tm_t_crosstable()
+
+  app_driver$expect_no_shiny_error()
+
+  testthat::expect_equal(
+    app_driver$get_text("#teal-main_ui-root-active_tab > li.active > a"),
+    "Cross Table"
+  )
+
+  encoding_dataset <- app_driver$get_text("#teal-main_ui-root-cross_table .help-block")
+  testthat::expect_match(encoding_dataset, "Dataset:\\n *ADSL\\n", all = FALSE)
+
+  app_driver$stop()
+})
+
```

Comment:
This test seems optional, given that we are no longer doing it on the other PRS (relying on example testing and initialization of other tests)

But I'm ok with keeping it if you feel it's necessary.

Reply:
Lets keep this  for now.